File: src/services/IsoManager.ts
import { IsoInfo } from '../types';

export class IsoManager {
  private static defaultPath = '/data/user/0/com.example.popvm/files/linux.iso';

  static getIso(): string {
    return this.defaultPath;
  }

  static getIsoInfo(): IsoInfo {
    return {
      name: 'linux.iso',
      path: this.defaultPath,
      size: '648 MB',
      arch: 'ARM64 (aarch64)',
      status: 'Attached'
    };
  }
}

File: app/src/main/java/com/example/popvm/services/IsoManager.kt
package com.example.popvm.services

import com.example.popvm.models.IsoInfo

object IsoManager {
    private const val DEFAULT_PATH = "/data/user/0/com.example.popvm/files/linux.iso"

    fun getIso(): String = DEFAULT_PATH

    fun getIsoInfo(): IsoInfo {
        return IsoInfo(
            name = "linux.iso",
            path = DEFAULT_PATH,
            size = "648 MB",
            arch = "ARM64 (aarch64)",
            status = "Attached"
        )
    }
}
